net/nheko/Portfile

```diff
@@ -4,26 +4,24 @@ PortSystem          1.0
 PortGroup           github 1.0
 PortGroup           cmake 1.1
 PortGroup           qt5 1.0
-PortGroup           cxx11 1.1
 PortGroup           compiler_blacklist_versions 1.0
 
-#NOTE: requires C++14
-
-github.setup        mujx nheko 0.5.2 v
+github.setup        nheko-reborn nheko 0.7.2 v
 categories          net chat
 platforms           darwin
 license             GPL-3
-maintainers         {@scarface-one gmail.com:sireeshkodali1}
+maintainers         {@scarface-one disroot.org:scarface}
 description         A matrix chat client
 long_description    Nheko is a native desktop matrix \
                     chat client. It aims to be more \
                     like a chat app and less like an \
                     IRC client
 
-checksums           rmd160  be32b10c51960a1d27dd4fa410f82e5dd1e98534 \
-                    sha256  d7c189efd04ffa032b46c5eb24cf076ade5ddec3434b937ec9d0de952bb8f79b \
-                    size    4526124
+checksums           rmd160  67e1790fabb3105c6ea19bdf04a2bfd4151b94a0 \
+                    sha256  0b31481df6aadcb040011247ebfcaf2bc4081b0be61edca627216a1d8fdc124d \
+                    size    899673
 
+compiler.cxx_standard  2014
 #C++14 was first fully supported by clang 3.4 aka apple clang 503.0.38
 # But TLS was first implemented by apple clang 800.0.38
 compiler.blacklist-append {clang < 800.0.38}
```

Comment:
As of MacPorts 2.6.3, this should be replaced with `compiler.thread_local_storage yes` (see https://trac.macports.org/wiki/CompilerSelection).

Reply:
Fixed, thanks!